sdk/lib/mirrors/mirrors.dart

```diff
@@ -174,11 +174,17 @@ external ClassMirror reflectClass(Type key);
  * If [key] is not an instance of [Type], then this function throws an
  * [ArgumentError].
  *
+ * One can optionally pass a list of [typeArguments] for a generic class.
```

Comment:
Thanks, this is very helpful! One question: would it be OK if I replace `type parameters` with `type variables` since that what's being used in Dart code?

I might need to update it to include generic typedefs as well (per @rmacnak-google  comments)

Unrelated, but is it supposed to be like this having "type parameters" in C and "type variables" in Dart referring to the same thing? It seems like unnecessary ambiguity to me, but maybe I'm not seeing the whole picture...